start.R

```diff
@@ -38,7 +38,7 @@ if (length(argv) > 0) {
 }
 
 # read in configuration
-cfg <- gms::check_config(cfg, reference_file="config/APT.cfg", modulepath = NULL)
+cfg <- gms::check_config(cfg, reference_file = cfg, modulepath = NULL)
```

Comment:
Honestly, what was the point of calling `gms::check_config()` in the first place?
> Checks a model configuration file for consistency by comparing it to a reference config file and the given module structure of the model. The function will throw out an error if settings are missing in the config which exist in the reference config, of if settings are set in the config which do not exist in the reference config file or if a realization is chosen for a module which does not exist, not allowed setting combinations.

Reply:
If I remember correctly.

Reply:
Translation:
> The function will throw out an error if settings are missing in `cfg` which exist in `cfg`, of if settings are set in `cfg` which do not exist in `cfg`.

Just cut it.

Reply:
But then you need all those settings in both `config/default.cfg` and `config/APT.cfg`.  For no other reason as to make a function designed for a completely different purpose work.

Reply:
> Translation:
> 
> > The function will throw out an error if settings are missing in `cfg` which exist in `cfg`, of if settings are set in `cfg` which do not exist in `cfg`.
> 
> Just cut it.

What does that mean?

Reply:
You mean I should simply `source(cfgFile)`? Maybe its better to still use `check_config`, it at least checks, if the file sourced has the structure of a config file as expected.

Reply:
No.  It checks whether `cfgFile` defines the same (no more, no less) settings as … `cfgFile`.  The function will never throw an error, no matter what you put into the .cfg file.
Its completely pointless.

Reply:
Ok, I replaced it with `source(cfg)`, agree?